### PR TITLE
384 - Label attributes issue

### PR DESCRIPTION
**Pull request by jeffwa428**
**Explain the details for making this change. What existing problem does the pull request solve?**

- Removed `.color-input-value-no-label` class in order to get rid of padding when there is no label.
- Added LABEL_AUDIBLE option to ids-checkbox
- Added new section for ids-checkbox with no label
- Updated `.audible` for `label-checkbox`

**Related github/jira issue (required)**:

Closes https://github.com/infor-design/enterprise-wc/issues/384

**Steps necessary to review your pull request (required)**:

- Go to demos/ids-color-picker/example.html and remove label attributes
- Go to http://localhost:4300/ids-color-picker and check if color picker ui should be displayed correctly without label

- Go to demos/ids-checkbox/example.html and add `label-audible` attribute with `true/false`
- Go to http://localhost:4300/ids-checkbox and check if checkbox should be displayed correctly without label

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
This is a great first PR! The only problem i see is pretty small.

Screen Shot 2021-11-18 at 9 44 05 AM

When no label it seems like the padding below the field is less than when there is one with a field. We should try and make that equal?

**Review by tmcconechy — APPROVED** [review]

**Comment by tmcconechy** [comment]
@jeffwa428 looks like a few tests to fix https://github.com/infor-design/enterprise-wc/runs/4253677698?check_suite_focus=true#step:9:246

`npx jest -- checkbox`

File: src/components/ids-checkbox/ids-checkbox.js [hunk 1]
```diff
@@ -325,6 +331,8 @@ class IdsCheckbox extends mix(IdsElement).with(
 
   get label() { return this.getAttribute(attributes.LABEL) || ''; }
 
+  get labelAudible() { return this.getAttribute(attributes.LABEL_AUDIBLE); }
+
```
Comment (andriivarhanov) [inline src/components/ids-checkbox/ids-checkbox.js:335]: please add `set` for `labelAudible` property

**Review by andriivarhanov — COMMENTED** [review]
looks good, couple of things:
- please add label audible to the readme with short description on what it does
- add label audible to typescript file `ids-checkbox.d.ts`
- add some e2e or func tests for label audible attribute and property

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Requesting changes. See @andriivarhanov comments please
Also please try to fix the tests https://github.com/infor-design/enterprise-wc/runs/4256625935?check_suite_focus=true#step:9:243 and check the coverage.

**Comment by EdwardCoyle** [comment]
@jeffwa428 This looks really nice now!  I just approved the percy tests as everything seems to be in order.  Could you address the outstanding comment(s?) from @andriivarhanov?  Then we can merge this 👍

**Comment by jeffwa428** [comment]
Okay, I almost finished that. I will push new update soon.

**Comment by jeffwa428** [comment]
@tmcconechy , @andriivarhanov @EdwardCoyle 
I have made changes requested and pushed. Kindly check it and let me know if there is any issue.

File: src/components/ids-checkbox/ids-checkbox.js [hunk 2]
```diff
@@ -325,6 +331,22 @@ class IdsCheckbox extends mix(IdsElement).with(
...
   get label() { return this.getAttribute(attributes.LABEL) || ''; }
 
+  /**
+   * Set the `label-audible` attribute
+   * @param {boolean} value of the `labelAudible`
+   */
+  set labelAudible(value) {
+    this.setAttribute(attributes.LABEL_AUDIBLE, value);
+    const idsTextElem = this.labelEl?.querySelector('ids-text');
+    if (value === 'true' || value === true) {
```
Comment (andriivarhanov) [inline src/components/ids-checkbox/ids-checkbox.js:341]: please use `IdsStringUtils.stringToBool` util here

**Review by andriivarhanov — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**